tests/tf/integration/test_ci_05_export_retrieval_model.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from testbook import testbook
+
+from tests.conftest import REPO_ROOT
+
+
+@testbook(REPO_ROOT / "examples/05-Retrieval-Model.ipynb", timeout=180, execute=False)
+def test_func(tb):
+    tb.inject(
+        """
+        import os
+        os.environ["DATA_FOLDER"] = "/raid/data/aliccp/raw/"
```

Comment:
is there a `raid` directory with raw alicpp dataset on CI machine?

Reply:
Yes should be in my understanding